src/exobot.js

```diff
@@ -116,25 +118,28 @@ export class Exobot extends Configurable {
   initPlugins(plugins = {}) {
     /* eslint no-param-reassign: 0 */
     const loadedPlugins = Object.keys(plugins).reduce((p, k) => {
-      const [, config] = plugins[k];
+      let [, config] = plugins[k];
```

Comment:
This could be let [plugin, config] = plugins[k]; now that we are using let instead of const.